packages/openapi-generator/README.md

```diff
@@ -31,47 +31,45 @@ USAGE
   $ generate-openapi-client --input <input> --outputDir <outputDirectory>
 
 OPTIONS
-  -i, --input=<path/to/input>                             (required) Specify the path to the directory or file
```

Comment:
I assume this is generated without manual adjustment.

Reply:
Correct.